pocs/focuser/focuslynx.py

```diff
@@ -28,6 +28,7 @@ class Focuser(AbstractFocuser):
     Additional positional and keyword arguments are passed to the base class, AbstractFocuser. See
     that class for a complete list.
     """
+
```

Comment:
Doesn't the args section belong in the docstring for __init__ rather than the class?

Reply:
There's not actually a preference according to the google style other than to say it should be consistent for a project. i think we have a lot of inconsistency in this regard. :(

Reply:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html